### PR TITLE
feat(menu): add menu exemple

### PR DESCRIPTION
**Pull request by pochretien**
add menu exemples

### Acceptance Criteria

-   [ ] The proposed changes are covered by unit tests
-   [ ] The potential breaking changes are clearly identified
-   [ ] [README.md](https://github.com/coveo/react-vapor/blob/master/README.md) is adjusted to reflect the proposed changes (if relevant)

### CODE REVIEW

**Comment by pochretien** [comment]
image

**Comment by pochretien** [comment]
Update the Menu with the Button component to allow Tooltip and Disabled state

File: packages/react-vapor/src/components/menu/MenuConnected.tsx [hunk 1]
```diff
@@ -49,12 +52,13 @@ const mapDispatchToProps = (dispatch: IDispatch, ownProps: IMenuOwnProps): IMenu
 @ReduxConnect(mapStateToProps, mapDispatchToProps)
 export class MenuConnected extends React.Component<IMenuProps, {}> {
     private list: HTMLDivElement;
-    private button: HTMLButtonElement;
+    private button: HTMLDivElement;
```
Comment (GermainBergeron) [inline packages/react-vapor/src/components/menu/MenuConnected.tsx]: It's not a button anymore tho 🤔

Comment (pochretien) [reply packages/react-vapor/src/components/menu/MenuConnected.tsx]: Nope, but I target the button with a div element so I keep the name button to explain what I want to calculate

**Pull request merged**